src/core/translation/locales/en/decentdispute.json

```diff
@@ -0,0 +1,287 @@
+{
+  "DecDispDisputes": "Disputes",
```

Comment:
I reviewed the PR and have a concern. If each page has its own specific JSON file for translations, and the keys in each file already start with the file name (to ensure uniqueness), why are we importing translations from individual files instead of importing them from a root file where all JSONs are combined into one object? Wouldn't this approach simplify the process and make it more consistent across the project?